MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewModel.swift

```diff
@@ -11,7 +11,8 @@ class MovieDetailsViewModel {
 
     // MARK: - Variables
     private let movieDetailsRepository: MovieDetailsRepositoryType
-    private var movieID: Int = 0
+    private let coreDataManager = CoreDataManager()
```

Comment:
viewModel should never know about core data manager.
Viewmodel should only know about repo. The repo knows about core data manager
Please adjust

Reply:
will adjust accordingly